code/modules/mob_spawn/ghost_roles/spider_roles.dm

```diff
@@ -34,7 +34,7 @@
 /obj/effect/mob_spawn/ghost_role/spider
 	name = "egg cluster"
 	desc = "They seem to pulse slightly with an inner life."
-	mob_name = "a spider"
+	mob_name = "\improper spider"
```

Comment:
if you don't mind them being called giant spider # 321  you can also just set this to null to achieve the same thing.

Reply:
Yeah, I know but i think it was intentional to just be `a spider`. If you want me to change it im fine with that